comm/p2p/manager.go

```diff
@@ -39,6 +43,10 @@ func (sm *StreamManager) ReleaseStreams(sessionID string) {
 	}
 
 	for peer, stream := range streams {
+		if stream.Conn() != nil {
+			_ = stream.Conn().Close()
+		}
```

Comment:
doesn't this mean that stream.Close beneath will always error?

Reply:
It seems that stream.Close() doesn't necessarily close the connection because other streams might have the same one.

Reply:
I get that, but it's the order that concerns me. If you first close conn, you won't be able to close stream as there is no conn to write closing messages